taxonomies/categories.txt

```diff
@@ -17708,6 +17865,14 @@ nl:Platte graanbroden, Platte graanbrood
 sv:Halvtjockkakor, Hällakaka
 wikidata:en:Q10516678
 
+<en:Wheat breads
+<en:Flatbreads
+en:Fajitas
```

Comment:
Fajitas is a meal wrapped in a tortilla (which is the flat bread). Fajitas should not be under flatbreads.

Reply:
OK I will remove <en:Flatbreads in the next PR. Thanks for the heads up!